lerna.json

```diff
@@ -6,22 +6,15 @@
     "command": {
         "publish": {
             "conventionalCommits": true,
-            "allowBranch": [
-                "master"
-            ]
+            "allowBranch": ["master"]
         },
         "version": {
             "conventionalCommits": true,
             "message": "chore(release): publish %s [skip travis]"
         },
         "bootstrap": {
             "hoist": true,
-            "nohoist": [
-                "unidiff",
-                "query-string",
-                "strict-uri-encode",
-                "split-on-first"
-            ]
+            "nohoist": ["unidiff", "query-string", "strict-uri-encode", "split-on-first"]
```

Comment:
Those are wanted changes ?

Reply:
Good question, I think that my IDE tends to format this file that way (and if I use prettier it does the same thing). When you change and push this file, does it usually look differently?

Reply:
Personnaly I would not commit this file in the PR since it is clearly out of scope :man_shrugging: